pkg/runtime/field_export_reconciler.go

```diff
@@ -340,6 +343,9 @@ func (r *fieldExportReconciler) writeToSecret(
 ) error {
 	// Construct the data key
 	key := fmt.Sprintf("%s.%s", desired.Namespace, desired.Name)
+	if desired.Spec.To.Key != nil && strings.TrimSpace(*desired.Spec.To.Key) != "" {
```

Comment:
Shall we handle cases where `Spec.To` is nil? to avoid nil panics?

Reply:
Done. f6caeceb92749758b5b6200c83dd384091650f8c